otter/test/models/test_cass_models.py

```diff
@@ -857,22 +875,16 @@ def test_modify_state_lock_not_acquired(self, mock_serial):
         ``modify_state`` raises error if lock is not acquired and does not
         do anything else
         """
-        self.lock.acquire.side_effect = \
-            lambda timeout: defer.fail(ValueError('a'))
+        self.acquire_call = (True, 10, Failure(ValueError("eh")))
 
         def modifier(group, state):
             raise
 
         self.group.view_state = mock.Mock(return_value=defer.succeed('state'))
-
         d = self.group.modify_state(modifier)
         self.failureResultOf(d, ValueError)
-
         self.assertEqual(self.connection.execute.call_count, 0)
-        self.kz_client.Lock.assert_called_once_with(
-            '/locks/' + self.group.uuid)
-        self.lock._acquire.assert_called_once_with(timeout=10)
-        self.assertEqual(self.lock.release.call_count, 0)
+        self.assertFalse(self.lock.acquired)
```

Comment:
`.acquired` is not a public attribute on the real implementation, so making this test more realistic would also break it.  https://glyph.twistedmatrix.com/2015/05/separate-your-fakes-and-your-inspectors.html is relevant here, I think.